biome.jsonc

```diff
@@ -8,7 +8,7 @@
 	"files": {
 		"ignoreUnknown": false,
 		"ignore": [],
-		"include": ["packages/**/*"]
+		"include": ["/packages/**/*"]
```

Comment:
_only_ files in ./packages (the leading "/" has the same meaning as in gitignore(5))